docs: add directions on opting out from global environment (#160)
danielroe/nuxt-vitest: README.md

})
```

If you have set `environment: 'nuxt'` by default, you can then opt-out [of the default environment](https://vitest.dev/guide/environment.html#test-environment) per test file as needed.

```js
// @vitest-environment node
import { test } from 'vitest'

test('my test', () => {
  // ... test without Nuxt environment!
})
```

> ## 👉 Important notes

Reviewer: ```suggestion

## 👉 Important notes
```

})
```

> And opt-out [to the default environment](https://vitest.dev/guide/environment.html#test-environment) per test file if the nuxt environment isn't needed.

Reviewer: ```suggestion
If you have set `environment: 'nuxt'` by default, you can then opt-out [of the default environment](https://vitest.dev/guide/environment.html#test-environment) per test file as needed.
```

```js
// @vitest-environment node
import { test } from 'vitest'

test('my test', () => {
  // ... test without Nuxt environment!
})
```

> 
> 

Reviewer: ```suggestion

```
